packages/compass-generative-ai/src/atlas-ai-service.ts

```diff
@@ -395,6 +467,60 @@ export class AtlasAiService {
     );
   }
 
+  /**
+   * @returns {MockDataSchemaResponse} which contains faker.js mappings used to produce a
+   * faker.js factory function for the purposes of generating mock document data.
+   *
+   * @throws {AtlasAiServiceApiResponseParseError} when the response cannot be parsed into the expected schema.
```

Comment:
This is confusing because that's not the only error type that can be thrown here, we don't really aim for documenting this, but if you want to add this documentation, let's add all possible error types here

Reply:
opted for removing it